charts/redpanda/Chart.yaml

```
@@ -23,7 +23,7 @@ type: application
 # The chart version and the app version are not the same and will not track
 # together. The chart version is a semver representation of changes to this
 # chart.
-version: 4.0.17
+version: 4.0.18
```

I think you need to bump to 
```suggestion
version: 5.0.0
```

It is because you will not be able to pass upgrade tests as you changes Stateful Set definition by adding volumes and volumeMounts